shell/apps/settings/locales/ca_ES.json

```diff
@@ -0,0 +1,233 @@
+{
+    "localeCode": "ca_ES",
+    "metadata": {
+        "name": "Català (Espanya)",
+        "nameShort": "Català",
+        "textDirection": "ltr"
+    },
+    "source": {
+        "settings": "Configuració",
+
+        "back": "Enrere",
+        "ok": "D'acord",
+        "cancel": "Cancel·la",
+        "apply": "Aplica",
+        "add": "Afegeix",
+        "save": "Desa",
+        "saving": "Desant...",
+        "remove": "Elimina",
+        "none": "(Cap)",
+        "optional": "(Opcional)",
+
+        "size_bytes": {
+            "{size} == 1": "{size} byte",
+            "{size} != 1": "{size} bytes"
+        },
+        "size_kb": "{size} KB",
+        "size_mb": "{size} MB",
+        "size_gb": "{size} GB",
+        "size_tb": "{size} TB",
+        "size_kib": "{size} KiB",
+        "size_mib": "{size} MiB",
+        "size_gib": "{size} GiB",
+        "size_tib": "{size} TiB",
+
+        "network": "Xarxa",
+
+        "network_summary": "Wi-Fi, Ethernet",
+        "network_summaryConnectedWifi": "Connectat a {name}",
+        "network_summaryConnectedEthernet": "Connectat via Ethernet",
+        "network_summaryDisconnected": "Desconnectat",
+
+        "network_connected": "Connectat",
+
+        "network_wifiNetworks": "Xarxes Wi-Fi",
```

Comment:
Here is where "Xarxes Wi-Fi" is currently used

Reply:
Yeah either way, so long as they're the same really, as it'll be what people search for in the app launcher